src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/instantiate/containers/index.tsx

```diff
@@ -94,57 +93,11 @@ function InstantiateContainer(props: IServiceInfoProps) {
           hideCloseIcon={true}
           headerText={titleText}
         >
-          <ContentLoader
-            height={200}
-          >
-            <rect
-              x='25' y='20'
-              rx='0' ry='0'
-              width='100' height='5'
-            />
-            <rect
-              x='25' y='30'
-              rx='0' ry='0'
-              width='350' height='5'
-            />
-            <rect
-              x='25' y='40'
-              rx='0' ry='0'
-              width='350' height='25'
-            />
-
-            <rect
-              x='25' y='75'
-              rx='0' ry='0'
-              width='100' height='5'
-            />
-            <rect
-              x='25' y='85'
-              rx='0' ry='0'
-              width='350' height='5'
-            />
-            <rect
-              x='25' y='95'
-              rx='0' ry='0'
-              width='350' height='25'
-            />
-
-            <rect
-              x='25' y='130'
-              rx='0' ry='0'
-              width='100' height='5'
-            />
-            <rect
-              x='25' y='140'
-              rx='0' ry='0'
-              width='350' height='5'
-            />
-            <rect
-              x='25' y='150'
-              rx='0' ry='0'
-              width='350' height='25'
-            />
-          </ContentLoader>
+          <div style={{ marginTop: '5rem', marginLeft: '8rem' }}>
```

Comment:
Prefer to have styling as a css class object that we refer to from the element, rather than directly on the html element. Use the existing `styles` object (l.21).
F.ex., add to the styles object:
```
contentLoaderContainer: {
  marginTop: '5rem',
  marginLeft: '8rem',
},
```
and in the div element,
```
<div className={classes.contentLoaderContainer}>
```

Reply:
Inlining styles can cause problems, later as, inlined styles have presidency over classes.

Reply:
Updated with a fix